test/graphemetest.c

```diff
@@ -49,7 +51,7 @@ int main(int argc, char **argv)
                 else
                     i++;
             }
-            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND);
+            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND | UTF8PROC_OLDENCODE);
```

Comment:
In general, it's good practice to merge patches only given regression tests and (where needed) documentation.